sample.mvn.parent/pom.xml

```diff
@@ -14,10 +16,43 @@
 	</parent>
 
 	<properties>
+		<!-- General Project Properties -->
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+		<!-- External Tools Properties -->
+		<bnd-bnd-maven-plugin-version>5.0.1</bnd-bnd-maven-plugin-version>
+		<m2e-lifecycle-mapping-version>1.0.0</m2e-lifecycle-mapping-version>
+
+		<!-- OSGi Properties -->
+		<osgi-org.osgi.service.component.annotations-version>1.3.0</osgi-org.osgi.service.component.annotations-version>
 		<osgi.qualifier>${maven.build.timestamp}</osgi.qualifier>
+
+		<!-- BestSolution/Efxclipse Properties -->
+		<bestsolution-maven-osgi-exec-plugin-version>0.0.3-SNAPSHOT</bestsolution-maven-osgi-exec-plugin-version>
+		<bestsolution-maven-osgi-package-plugin-version>0.0.3-SNAPSHOT</bestsolution-maven-osgi-package-plugin-version>
+		<efxclipse.rt-org.eclipse.equinox.executable-version>3.8.100-SNAPSHOT</efxclipse.rt-org.eclipse.equinox.executable-version>
+		<org.eclipse.fx.ui.workbench.fx.application-version>1.0.0</org.eclipse.fx.ui.workbench.fx.application-version>
+
+		<!-- OpenJFX Properties -->
+		<openjfx-javafx-controls-version>11</openjfx-javafx-controls-version>
+		<openjfx-openjfx.base-version>11.0.0</openjfx-openjfx.base-version>
+		<openjfx-openjfx.controls-version>11.0.0</openjfx-openjfx.controls-version>
+		<openjfx-openjfx.graphics.linux_64-version>11.0.0</openjfx-openjfx.graphics.linux_64-version>
+		<openjfx-openjfx.graphics.mac-version>11.0.0</openjfx-openjfx.graphics.mac-version>
+		<openjfx-openjfx.graphics.win32_64-version>11.0.0</openjfx-openjfx.graphics.win32_64-version>
+		<openjfx-openjfx.standard.feature-version>11.0.0</openjfx-openjfx.standard.feature-version>
+
+		<!-- Maven Properties -->
 		<maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<equinox-maven-plugin-version>0.0.3-SNAPSHOT</equinox-maven-plugin-version>
+		<maven.plugins-maven-compiler-plugin-version>3.8.0</maven.plugins-maven-compiler-plugin-version>
+		<maven.plugins-maven-compiler-plugin.source>11</maven.plugins-maven-compiler-plugin.source>
+		<maven.plugins-maven-compiler-plugin.target>11</maven.plugins-maven-compiler-plugin.target>
+		<maven.plugins-maven-jar-plugin.version>3.1.0</maven.plugins-maven-jar-plugin.version>
+
+		<!-- Tycho Properties -->
+		<tycho-target-platform-configuration-version>1.2.0</tycho-target-platform-configuration-version>
```

Comment:
should I consolidate this one as tycho-version? in the maven repositories they're always matching versions. The last one I tried is 1.6.0

Reply:
yes and updating to the latest tycho makes sense, once we release the next version of efxclipse it is a requirement to use a version > 1.5.x